@kiva/kv-components/vue/KvLendCta.vue

```diff
@@ -54,10 +54,54 @@
 		>
 			<fieldset
 				class="tw-w-full tw-flex"
+				:class="{'tw-flex-col tw-gap-1': showPresetAmounts}"
 				:disabled="isAdding"
 				data-testid="bp-lend-cta-select-and-button"
 			>
-				<div class="amountDropdownWrapper">
+				<div
+					v-if="showPresetAmounts"
+					class="tw-flex tw-gap-1"
+				>
+					<template v-if="!showUnreservedAmountOnly">
+						<kv-ui-button
+							v-for="option in defaultAmountOptions"
+							:key="option"
+							:state="`${ unreservedAmount < option ? 'disabled' : ''}`"
+							variant="secondary"
+							class="tw-inline-flex tw-flex-1 preset-option"
+							:class="{'selected-option': selectedButtonOption === option }"
+							data-testid="bp-lend-cta-lend-button"
+							type="submit"
+							@click="selectedButtonOption = option"
```

Comment:
It looks like these buttons are missing new analytics. Please make sure analytics similar to the dropdown are logged on click.